types/craftos/craftos.d.ts

```diff
@@ -6,128 +6,138 @@ type Color = number;
 type Colour = Color;
 
 declare namespace colors {
-    var white: Color;
```

Comment:
Please revert the reformatting. This makes for a huge commit when it's not necessary, and it goes against my personal code style.

Reply:
Sorry About the reformatting, I'm running the Prettier Plugin for vs code and the formatting was automatic. the Formatting is consistent with itself, but it's not my code base so fair enough I'll revert it.